enterprise_gateway/services/kernelspecs/handlers.py

```diff
@@ -1,17 +1,85 @@
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
 """Tornado handlers for kernel specs."""
-
+import json
 import notebook.services.kernelspecs.handlers as notebook_handlers
 import notebook.kernelspecs.handlers as notebook_kernelspecs_resources_handlers
+from notebook.utils import maybe_future
+from notebook.services.kernelspecs.handlers import is_kernelspec_model, kernelspec_model
 
+from tornado import web
 from ...mixins import TokenAuthorizationMixin, CORSMixin, JSONErrorsMixin
+from ...base.handlers import APIHandler
 
 # Extends the default handlers from the notebook package with token auth, CORS
 # and JSON errors.
 default_handlers = []
-for path, cls in notebook_handlers.default_handlers:
-    # Everything should have CORS and token auth
```

Comment:
Could you please preserve (and update) this comment at the end of file?  That code isn't very intuitive and I've found this helpful.  By 'update' I mean something like: `# Everything should have CORS, token auth, and JSON error handling` - thanks.